dashboard/src/components/AppView/AccessURLItem.scss

```diff
@@ -0,0 +1,13 @@
+.ServiceItem__url a {
```

Comment:
Maybe these two can be in the same one, `a` being inline.

Reply:
+1, also I think there should be some shared properties here. Maybe this should be:

```
.ServiceItem {
  display: inline-block;
  color: #fff;
  border-radius: 16px;
  &__url {
    background-color: #1598cb;
  }
  &__not-url {
    background-color: #b3b3b3;
  }
}
```

Also just to check, are these colours from the Hex palette?